lib/amazon.js

```diff
@@ -188,3 +197,17 @@ exports.createLambdaRole = async function (name) {
 
   return result.Role.Arn
 }
+
+exports.uploadToS3 = async function ({ zipFile, functionName, bucketName }) {
+  const key = `${functionName}-${revHash(zipFile)}.zip`
+
+  const params = {
+    Body: zipFile,
+    Bucket: bucketName,
+    Key: key
+  }
+
+  await s3.putObject(params).promise()
+
+  return key
```

Comment:
Not a biggie, but would prefer reading the key out from the response. That way the code ~~is~~ looks more functional than having functions that modify global state 😄

Reply:
Do you get it in the response? 🤔 I will check.

Reply:
I can't find the key in the response

Reply:
😞 oh well